.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +12,17 @@
 
 **PR Content:**
 
-* 
+**What type of change are you bringing in this PR?**
+- [ ] Backend changes
+- [ ] UI changes
+- [ ] Other
+
+**If possible, what kind of change it is (in detail)?**
```

Comment:
Can you change this to say 'Explain your changes in detail'?
I want people to put in some effort to explain their PR content